### PR TITLE
:bookmark: (24.8.0)

### PR DESCRIPTION
**Pull request by MatissJanis**
- web: https://github.com/actualbudget/actual/pull/3179
- server: https://github.com/actualbudget/actual-server/pull/408
- docs: https://github.com/actualbudget/docs/pull/421

### CODE REVIEW

File: blog/2024-08-02-release-24.8.0.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+date: 2024-08-03T10:00
+slug: release-24.8.0
+tags: [announcement, release]
+hide_table_of_contents: false
+authors: MatissJanis
+---
+
+The release has the following notable improvements:
+
+- TODO
```
Comment (MatissJanis) [inline blog/2024-08-02-release-24.8.0.md:13]: Any suggestions what to highlight?

Comment (twk3) [reply blog/2024-08-02-release-24.8.0.md:13]:
```
- [#2814](https://github.com/actualbudget/actual/pull/2814) Shows favourite and up to the top 5 most frequently used payees in the payee dropdown menu in a section at the top. — thanks @qedi-r
- [#3012](https://github.com/actualbudget/actual/pull/3012) Add long term goal type template — thanks @youngcw
- [#3061](https://github.com/actualbudget/actual/pull/3061) Add additional keyboard hotkeys. — thanks @psybers
- [#3132](https://github.com/actualbudget/actual/pull/3132) In spending report - adding last month as an option for the primary graph — thanks @carkom
```

These are the ones that strike me as positively impacting a broad section of the users

Comment (MatissJanis) [reply blog/2024-08-02-release-24.8.0.md]: Added most of these! Thanks for the suggestion.

File: blog/2024-08-02-release-24.8.0.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+slug: release-24.8.0
+tags: [announcement, release]
+hide_table_of_contents: false
+authors: MatissJanis
+---
+
+The release has the following notable improvements:
+
+- ability to favorite payees
+- most used an favorite payees appear at the top of the payee autocomplete results
```
Comment (matt-fidd) [inline blog/2024-08-02-release-24.8.0.md:14]: an -> and

**Review by matt-fidd — CHANGES_REQUESTED** [review]

**Review by matt-fidd — APPROVED** [review]

**Pull request merged**